cmd/completion.go

```diff
@@ -0,0 +1,224 @@
+package cmd
+
+import (
+	"bytes"
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+var shell string
+
+func MakeCompletion() *cobra.Command {
+	var completionCmd = &cobra.Command{
+		Use:   "completion SHELL",
+		Short: "Generates shell completion",
+		Long: `Generates shell auto completion for Bash or ZSH.
+		
+Please follow the instructions in the link below to activate the shell auto completion in your environment:
+https://docs.openfaas.com/cli/completion/`,
```

Comment:
I think this link is not correct.

Reply:
I took this snippet from faas-cli project, so users could have a little bit of a context on how to set up the completion.
I could include these instructions on the link to the readme.md, and remove this from the completion command help.
Sounds good?